python-sdk/src/astro/settings.py

```diff
@@ -31,3 +31,9 @@
 LOAD_TABLE_AUTODETECT_ROWS_COUNT = conf.getint(
     section="astro_sdk", key="load_table_autodetect_rows_count", fallback=1000
 )
+
+
+#: Reduce responses sizes returned by aql.run_raw_sql to avoid trashing the Airflow DB if the BaseXCom is used.
+RAW_SQL_MAX_RESPONSE_SIZE = conf.getint(
+    section="astro_sdk", key="run_raw_sql_response_size", fallback=-1
```

Comment:
```suggestion
    section="astro_sdk", key="run_raw_sql_response_size", fallback=0
```
WDYT of making it consistent with `response_limit` i.e. setting it to `0`?

Reply:
@feluelle I find the proposed change more confusing to users. It seems more intuitive that `response_size` set to 0 means to return zero results, as opposed to disabling `response_size` checking. The -1 gives clarity: that we are not checking `response_size`.

But I'm all for consistency, so once we agree - I can change the behaviour in the correct place

Reply:
ATM, I can't think of an obvious situation when the user would like to set `response_size=0`, but I believe it is a fair assumption that if a user did this, they'd expect the decorator to return zero results, as opposed to not limit the response size.

I'd be happy with using None to represent no response to the code, including: https://github.com/astronomer/astro-sdk/blob/e5bc9bb7ca1de34e789c5441baa7091877e22de0/python-sdk/src/astro/settings.py#L37-L39
Do you think this change is worth the additional complexity?

Reply:
@tatiana @feluelle For me, -1 is better than explicitly saying to return all results and 0 means don't send any response back.